frontend/src/app/shared/components/editor/components/ckeditor/ckeditor-setup.service.ts

```diff
@@ -99,11 +104,26 @@ export class CKEditorSetupService {
   /**
    * Load the ckeditor asset
    */
-  private static load():Promise<unknown> {
+  private static async load():Promise<string> {
     // untyped module cannot be dynamically imported
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
-    return import(/* webpackChunkName: "ckeditor" */ 'core-vendor/ckeditor/ckeditor.js');
+    await import(/* webpackChunkName: "ckeditor" */ 'core-vendor/ckeditor/ckeditor.js');
+
+    /** If we're on the english locale, there is no file to load */
+    if (I18n.locale === 'en') {
+      return 'en';
+    }
+
+    try {
+      await import(
+        /* webpackChunkName: "ckeditor-translation" */ `../../../../../../vendor/ckeditor/translations/${I18n.locale}.js`
+      ) as unknown;
+      return I18n.locale;
+    } catch (e:unknown) {
+      console.warn(`Failed to load translation for CKEditor: ${e as string}`);
+      return 'en';
```

Comment:
It's a bit weird that an editor `load` function returns a locale name string. I'd probably just set the locale in this method.

Reply:
Yeah good point, I'll change that.